src/ast/DirDecl.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Represents the directories a TypeScript project may have.
+ *
+ * e.g. dirs [src/model/model, test/util]
+ */
+export class DirDecl {
```

Comment:
should these extend an abstract class Node?
and parse evaluate functions throughout?